pages/common/git-lock.md

```diff
@@ -0,0 +1,9 @@
+# git lock
+
+> Lock git from following changes of a local file.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-lock>.
+
+- Disable the ability to commit changes of a local file:
```

Comment:
Does it also prevent from pulling changes from remote?

Reply:
No